README.md

```diff
@@ -1,8 +1,55 @@
-# Open 3D Engine Documentation & Community Special Interest Group
+# O3DE Documentation & Community Special Interest Group
 
-Materials for the Open 3D Engine Foundation Documentation & Community Special Interest Group (O3DE D&C SIG).
-We run the [docs.o3de.org](http://docs.o3de.org) website, manage its content, and work with the community to
-set standards and keep O3DE a collaborative, accepting environment for all.
+Welcome to the Documentation and Community Special Interest Group (D&C SIG) of Open 3D Foundation! 
 
-* [O3DE D&C SIG Charter](governance/charter.md)
-* [O3DE D&C SIG Meeting Notes](meetings/README.md)
+We run the [o3de.org/docs](http://o3de.org/docs) website, manage its content, and work with the Open 3D Engine (O3DE) community to set standards and keep O3DE a collaborative, accepting environment for all. The SIG consists of a docs side and a community side who work closely together to provide learning content and community events. Just to name a few, this includes O3DE Documentation, video tutorials, blog posts, and live streams.
+
+You can reach us at @sig-docs-community on Discord!
+
+For a broader view of the whole O3DF community, go to the [o3de/community](https://github.com/o3de/community/) repo.
```

Comment:
Other O3DF projects will run out outside of our governance/community model - keep it specific for now.

```suggestion
For a broader view of the whole O3DE community, go to the [o3de/community](https://github.com/o3de/community/) repo.
```